rustls/tests/api.rs

```diff
@@ -2931,6 +2932,14 @@ impl KeyLog for KeyLogToVec {
     }
 }
 
+impl Debug for KeyLogToVec {
```

Comment:
Why not derive it?

Reply:
We don't want to include `client_random` and `secret`. I'll add a comment indicating that to `fmt`.

Reply:
It's just test code, though... so not that important?

Reply:
Fair point. I'll switch to deriving the trait.

Reply:
Fixed.